core/services/ardupilot_manager/mavlink_proxy/MAVLinkRouter.py

```diff
@@ -55,8 +61,8 @@ def binary_name() -> str:
     def _validate_endpoint(endpoint: Endpoint) -> bool:
         return endpoint.connection_type in [
             EndpointType.UDPServer,
-            EndpointType.UDPClient,
```

Comment:
Why UDPClient is not here, it's the `-e`, you can check that with `nc`

Reply:
I forgot to change there since our last talk.
Just to confirm, we concluded that the `-e` creates an output UDPClient, and is not possible to use UDPServer as output, right?

Reply:
yes, you can test with netcat.

Reply:
Yeah. I get somewhat confused with netcat syntax. Will take a better look at it's docs.

Reply:
```
pacman -S tldr
tldr nc
```